README.md

```diff
@@ -144,15 +153,24 @@ those rules whenever possible.
 
 ## [](#creators)Creators
 
-**Christian Aichner**
-
-- <https://twitter.com/realaichner>
-- <https://www.facebook.com/aichner.christian>
-- <https://github.com/Aichnerc>
-
-**Florian Kleber**
-
-- <https://github.com/kleberbaum>
+<table border="0">
+	<tr>
+		<td>
+		<a href="https://github.com/schettn">
+			<img src="https://avatars.githubusercontent.com/schettn?s=100" alt="Avatar schettn">
+		</a>
+		</td>
+    <td>
+      <a href="https://github.com/kleberbaum">
+        <img src="https://avatars.githubusercontent.com/kleberbaum?s=100" alt="Avatar kleberbaum">
+      </a>
+    </td>
+    </tr>
+    <tr>
+    	<td><a href="https://github.com/schettn">Nico Schett</a></td>
+    	<td><a href="https://github.com/kleberbaum">Florian Kleber</a></td>
+    </tr>
+</table>
```

Comment:
I think it would be better that way.
```suggestion
<table border="0">
    <tr>
        <td>
    	    <a href="https://github.com/kleberbaum">
    	        <img src="https://avatars.githubusercontent.com/kleberbaum?s=100" alt="Avatar kleberbaum">
            </a>
        </td>
    	<td>
    	    <a href="https://github.com/schettn">
                <img src="https://avatars.githubusercontent.com/schettn?s=100" alt="Avatar schettn">
            </a>
    	</td>
    </tr>
    <tr>
        <td><a href="https://github.com/kleberbaum">Florian Kleber</a></td>
        <td><a href="https://github.com/schettn">Nico Schett</a></td>
    </tr>
</table>
```